locksmith/migrations/20220420111523-add-recipients-field-to-payment-intent.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('PaymentIntents', 'recipients', {
+      type: Sequelize.ARRAY(Sequelize.STRING),
```

Comment:
Let's use the `Sequelize.JSON` instead for compliance.

Reply:
okay